server/src/api/resolvers/mutation.ts

```diff
@@ -313,6 +343,9 @@ export const Mutation = {
     createBudget,
     updateBudget,
 
-    createPayment,
-    createApproval
+    createPayment: createPayment,
+    createApproval: createApproval,
+
+    createLineItem: createLineItem,
+    updateLineItem: updateLineItem
```

Comment:
For these, you don't need to repeat the variable name in the latest version of JS, so you can just do like 
```
createPayment,
createApproval
...
```
Like how it's done above